<ui version="4.0">
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQCompartmentsWidget.ui,v $ </comment>
<comment>   $Revision: 1.2 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2009/05/04 18:18:59 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <class>CQCompartmentsWidget</class>
 <widget class="CopasiWidget" name="CQCompartmentsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>456</height>
   </rect>
  </property>
  <property name="caption" stdset="0">
   <string>Compartments</string>
  </property>
  <layout class="QGridLayout">
   <item row="8" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnDelete">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnClear">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Delete All</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRight">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QTableView" name="mpTblCompartments">
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="3" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Filter Expression:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mpLEFilter"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasiwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpTblCompartments</tabstop>
  <tabstop>mpBtnDelete</tabstop>
  <tabstop>mpBtnClear</tabstop>
 </tabstops>
 <includes>
  <include location="global">vector</include>
  <include location="local">copasi/UI/copasiWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnClear</sender>
   <signal>pressed()</signal>
   <receiver>CQCompartmentsWidget</receiver>
   <slot>slotBtnClearClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDelete</sender>
   <signal>pressed()</signal>
   <receiver>CQCompartmentsWidget</receiver>
   <slot>slotBtnDeleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpTblCompartments</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>CQCompartmentsWidget</receiver>
   <slot>slotDoubleClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
